examples/backtrace_example.cpp

```diff
@@ -0,0 +1,60 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
I'm a bit unclear what the example is supposed to be showing. Can you clarify (in reply to this is fine).

Reply:
`alloc_leak_example` prints a backtrace for each active allocation.  Further, that memory is leaked because it was never free.  It will also show that the resource manager logs the leaked allocations along with backtraces for who allocated them

`umpire_exception_example` shows that a backtrace is included with the thrown exception string returned by `exc.what()`.

Reply:
@davidbeckingsale, may I mark this conversation resolved?